debug/parse_elevator_text.py

```diff
@@ -0,0 +1,44 @@
+import json

```

Comment:
Utility function I want to keep around, useful for testing rooms and adding new ones for start room